src/js/table.js

```diff
@@ -0,0 +1,235 @@
+(function (Wood) {
+    Wood.Table = Marionette.ItemView.extend({
```

Comment:
I haven't touched the Table code, that is a TODO, however, my plan is to just have a base table class that takes a collection as an argument